stdlib/private/StdlibUnittest/StdlibUnittest.swift.gyb

```diff
@@ -1180,6 +1187,10 @@ public enum TestRunPredicate : CustomStringConvertible {
     case LinuxAny(reason: let reason):
       return "LinuxAny(*, reason: \(reason))"
     }
```

Comment:
This brace terminates the switch statement, it shouldn't be present.  Does the code build for you?  It fails for me.